src/test/java/org/lobid/resources/Hbz01MabXml2ElasticsearchLobidTest.java

```diff
@@ -128,7 +128,11 @@ static void etl() {
 		opener.process(
 				new File(Hbz01MabXmlEtlNtriples2Filesystem.TEST_FILENAME_ALEPHXMLCLOBS)
 						.getAbsolutePath());
-		opener.closeStream();
+		try {
+			opener.closeStream();
+		} catch (Exception e) {
+			// ignore, see #1030
```

Comment:
This will catch and ignore anything that could go wrong here, right?

We could catch ` NullPointerException` only to narrow it down a bit.

Is there way to test here if we actually have more threads than documents?

And I'd suggest we use the full issue URL (`https://github.com/hbz/lobid-resources/issues/1030`) for #1030 to make it unambiguous and easier to access from the code.

Reply:
Added a commit to catch NPE and use URI as reference in the comment.
> Is there way to test here if we actually have more threads than documents?

I wouldn't know of that, because the number of processed documents will only be known when the etl is finished, while the threads are already set. One could think of a preprocessing pipeline, but what would be the point?  I think the real fix would be to somehow make the metafacture library more robust like mentioned in #1030.